Gemfile

```diff
@@ -14,4 +14,4 @@ gem 'kitchen-verifier-shell', '~> 0.2.0'
 gem 'kitchen-verifier-serverspec', '~> 0.3.0'
 gem 'infrataster', '~> 0.3.2', :git => 'https://github.com/trombik/infrataster.git', :branch => 'reallyenglish'
 gem 'serverspec', '~> 2.37.2'
-gem 'specinfra', '>= 2.63.2' # OpenBSD's `port` is fixed in this version
+gem 'specinfra', '>= 2.66.6'
```

Comment:
when you changes files that `qansible` creates by default, and the modification should be applied to all roles, please create an issue in `qansible` repository. `qansible qa` warns when `Gemfile` is not identical with the default `Gemfile`, and this ensures certain, and same, level of quality in all roles.